pkg/controller/control.go

```diff
@@ -1250,19 +1254,27 @@ func (c *Controller) handleDiskNoSpaceErrorForReplicas(replicaNoSpaceErrMap map[
 	}
 
 	replicaWritableCount := 0
+	writableReplicaNoSpaceList := []string{}
```

Comment:
NIT: Should not this be `readWriteReplicaNoSpaceList`? The function does not check and count WO replicas